docs/fendermint/general_cross_messages.md

```diff
@@ -0,0 +1,120 @@
+## Overview
+
+General cross messages enable the transmission of arbitrary messages between IPC subnets. They are particularly useful for transferring tokens (when supported, more on this later) or transmitting arbitrary data. This is not a feature that allows users to trigger messages from wallets; rather, it is a framework for cross-subnet contract-to-contract communication.
```

Comment:
But also account to contract communication? A wallet _can_ send a transfer to an account or contract on another subnet, and it travels through the same mechanism as described here.

Reply:
Only smart contracts can trigger `sendContractXnetMessage`. While a wallet can interact with a smart contract that makes this call, it would be inaccurate to imply a direct invocation from a wallet.

Reply:
AFAIK the intention of this document is to describe the General Message Passing rails and APIs, not just the Call message type, right? Wallets can call `Gateway#release` with no restrictions, which under the hood emits a `Transfer` type cross-net message, which in turn results in a `Result` response eventually, right? My impression was that this doc is not restricted to what's downstream of `sendContractXnetMessage` only.

Reply:
Yes, that's because these documents are intended for L3+ subnets, and currently the release and fund actions are limited to one level of communication (parent to child and vice versa). I will add a mention about this.